tests/conftest.py

```diff
@@ -54,15 +109,28 @@ def app():
         yield create_app()
 
 
-@pytest.fixture()
-def flask_test_client():
+def _flask_test_client(user_token=None):
     """
     Creates the test client we will be using to test the responses
     from our app, this is a test fixture.
     :return: A flask test client.
     """
-    with create_app().test_client() as test_client:
-        yield test_client
+    with create_app().app_context() as app_context:
+        with app_context.app.test_client() as test_client:
+            test_client.set_cookie("localhost", "fsd_user_token", user_token)
+            yield test_client
+
+
+@pytest.fixture(scope="function")
+def flask_test_client():
+    valid_user_token = create_valid_token()
+    return next(_flask_test_client(user_token=valid_user_token))
+
+
+@pytest.fixture(scope="function")
+def flask_test_client_invalid_token():
```

Comment:
do we use this anywhere?

Reply:
Ah meant to add a test for this. I've now added an additional test but removed this fixture and just updated the cookie in the test when it's called instead 👍